web/src/main/java/com/futurewei/alcor/web/entity/vpc/VpcWebRequestObject.java

```diff
@@ -43,7 +43,7 @@
     private Integer mtu;
 
     @JsonProperty("port_security_enabled")
-    private boolean portSecurityEnabled = true;
```

Comment:
Is portSecurityEnabled true by default in Neutron?

Reply:
I checked openstack, it shows that its valid values are enabled (true) and disabled (false). This value is used as the default value of port_security_enabled field of a newly created port. I think the default value is decided by ourself

Reply:
We should decide the value but also need to know the Neutron behavior. Please check with @cj-chung to see if he could help here.